tests/testthat/output/trunc_mat/newline.txt

```diff
@@ -0,0 +1,5 @@
+# A tibble: 2 x 1
+   `\n`
+  <chr>
+1  "\n"
```

Comment:
@hadley: Okay to use quotes in the printed output only if necessary (like with some CSV formats)?